pod/video/tests/test_models.py

```diff
@@ -942,3 +943,75 @@ def test_delete_UserMarkerTime(self):
         markerTime.delete()
         self.assertEqual(UserMarkerTime.objects.all().count(), 0)
         print(" ---> test_delete_UserMarkerTime: OK!")
+
+
+class VideoAccessTokenTestCase(TestCase):
+    """Test the VideoAccessToken model."""
+
+    fixtures = [
+        "initial_data.json",
+    ]
+
+    def setUp(self):
```

Comment:
Attention : setup est lancé avant chaque test. Pour n'ajouter qu'une vidéo, je recommande d'ajouter un attribut de classe `_one_time_setup_complete = False`qu'on passe à true une fois le setup fait, et ne pas faire le contenu du setup que si `_one_time_setup_complete is False`

Reply:
Je ne suis pas certain car si tu fais un print du nombre de video créées, il est à zero à chaque passage en setup